src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -21,258 +21,120 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
     dbFutureValue { _.clusterQuery.list() } shouldEqual Seq()
 
     lazy val err1 = ClusterError("some failure", 10, Instant.now().truncatedTo(ChronoUnit.SECONDS))
-    lazy val c1UUID = UUID.randomUUID()
+    lazy val cluster1UUID = UUID.randomUUID()
     val createdDate, dateAccessed = Instant.now()
-    val c1 = Cluster(
-      clusterName = name1,
-      googleProject = project,
-      serviceAccountInfo = ServiceAccountInfo(Some(serviceAccountEmail), Some(serviceAccountEmail)),
-      dataprocInfo = DataprocInfo(Option(c1UUID), Option(OperationName("op1")), Some(GcsBucketName("testStagingBucket1")), Some(IP("numbers.and.dots"))),
-      auditInfo = AuditInfo(userEmail, createdDate, None, dateAccessed),
-      machineConfig = MachineConfig(Some(0),Some(""), Some(500)),
-      clusterUrl = Cluster.getClusterUrl(project, name1, clusterUrlBase),
-      status = ClusterStatus.Unknown,
-      labels = Map("bam" -> "yes", "vcf" -> "no"),
-      jupyterExtensionUri = None,
-      jupyterUserScriptUri = None,
-      errors = List.empty,
-      instances = Set(masterInstance, workerInstance1, workerInstance2),
-      userJupyterExtensionConfig = Some(userExtConfig),
-      autopauseThreshold = if (autopause) autopauseThreshold else 0,
-      defaultClientId = None
-    )
+    val cluster1 = getCluster(1).copy(dataprocInfo = getDataprocInfo(1).copy(googleId = Some(cluster1UUID)),
+                                      auditInfo = auditInfo.copy(createdDate = createdDate, dateAccessed = dateAccessed),
+                                      instances = Set(masterInstance, workerInstance1, workerInstance2))
```

Comment:
It'd be nice to align dataprocInfo, auditInfo and instances. Same for the copy(...) usages below.

Reply:
They were unaligned on my IntelliJ editor without doing anything. From my experience, IntelliJ tends to shuffle things around *once* you make modifications but not otherwise so I'm not sure why you may be having that issue.